quickstarts/spring-with-s2i.yaml

```diff
@@ -0,0 +1,208 @@
+  introduction: >-
+    **Spring** is a Java framework for building applications based on a distributed microservices architecture. 
+    
+    - Spring enables easy packaging and configuration of Spring applications into a self-contained executable application which can be easily deployed as a container to OpenShift.
```

Comment:
`Spring enables easy packaging and configuration of Spring applications` doesn't sound right